cmd/init_project.go

```diff
@@ -95,8 +95,8 @@ func (o *projectOptions) bindCmdlineFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.dep, "dep", true, "if specified, determines whether dep will be used.")
 	o.depFlag = cmd.Flag("dep")
 	cmd.Flags().StringArrayVar(&o.depArgs, "depArgs", nil, "Additional arguments for dep")
-	cmd.Flags().MarkDeprecated("dep", "use the fetch-deps flag instead")
-	cmd.Flags().MarkDeprecated("depArgs", "will be removed with version 1 scaffolding")
+	_ = cmd.Flags().MarkDeprecated("dep", "use the fetch-deps flag instead")
```

Comment:
Why it `_` would be required here?

Reply:
This explicitly "handles" the error by not doing anything. It is recommended by some IDEs and I find it a good way to show that there is something being returned that we are not using.